setup.cfg

```diff
@@ -69,7 +69,8 @@ all =
     tensorflow > 1.10, < 2.3
     ; Version capped due to tensorflow incompatibility
     protobuf < 4
-    matplotlib
+    ; Version capped due to shap incompatibility
+    matplotlib < 3.6.0
```

Comment:
Note that whatever change you make here, the same change should also be made to the `plt` extra.